drivers/modules/resources/partials/http-api/responses/query.json.adoc

```diff
@@ -0,0 +1,9 @@
+[source,json]
+----
+{
+  "queryType": "read" | "write" | "schema",
+  "answerType": "ok" | "conceptRows" | "conceptDocuments",
+  "answers": [ ... ], // optional
```

Comment:
So will this polymorphically be either `rows` or `documents`?
i wonder if we should consider having `rows: nullable` or `documents: nullable` (or use empty lists)?

Reply:
i suppose that's true... ok